cmd/tk/env.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"sort"
```

Comment:
I think this branch needs a rebase, there is already some sorting here (from #521)